sources/TagCheckerPI.cpp

```diff
@@ -164,18 +164,16 @@ bool DoIDTree(bool perform_fix) {
 
 //*****************************************************************************
 bool DoAttributes(bool perform_fix) {
-  ProcessStructureElementFunction attributes = [](bool perform_fix, PDSElement element, 
+  ProcessStructureElementFunction attributes = [](bool perform_fix, PDSElement element,
     ASAtom kid_type, PDSElement kid, PDSMCInfo mcid_info, PDSMC marked_content) {
-    if (kid_type == ASAtomFromString("StructElem")) {
-      CosObj kid_obj = PDSElementGetCosObj(kid);
```

Comment:
PDSElement kid je v skutocnosti vzdy CosObj
NPROC (ASAtom, PDSElementGetKidWithMCInfo, (PDSElement element,
                                            ASInt32 index,
                                            CosObj* cosObjKid,
                                            PDSMCInfoP mcidInfo,
                                            void** pointerKid,
                                            CosObj* cosPage))

Reply:
este doplnim, ze je to CosObj* lebo to moze byt SE, MC alebo MCR a preto sa to muselo kontrolovat. Po novom sa v tych upravenych callbackoch nepouziva ten kid, ale uz PDSElement element, co je vzdy len SE